tests/main/classic-ubuntu-core-transition-two-cores/task.yaml

```diff
@@ -36,7 +36,8 @@ execute: |
 
     while ! snap changes|grep ".*Done.*Transition ubuntu-core to core"; do
         snap changes
-        snap change 3||true
+        chg_id=$(snap changes|grep -o -P "^\d+(?= .*Transition ubuntu-core to core)" || true)
```

Comment:
Given that we use this helper 3 times currently, I wonder if we just make it a shell helper function in `tests/lib/something.sh` (changes.sh, snaps.sh?).